.dockerignore

```diff
@@ -0,0 +1,6 @@
+**/.git
+**/.github
+**/.gitignore
+**/Dockerfile
+**/*.Rproj
+docker-compose.yml
```

Comment:
Given that local testing is a common pattern with this repo, with some very large files that could get baked in, I'm thinking that a preferable pattern for this dockerignore is to exclude _everything_ and then add back in the few files that we do want to include.

```suggestion
# ignore everything
*
# include certain files
!config.yml
!DESCRIPTION
!Dockerfile
!LICENSE
!LICENSE.md
!README.md
!run_pacta_data_preparation.R
```

Reply:
personally, I don't target input or output directories in/under the root directory when using locally, but fair enough, totally ok with that strategy

however...
- is it necessary/desirable to (explicitly) include the Dockerfile inside the Docker image?
- I believe it is necessary to include the `.env` file (because the desired config set is specified there, not just for the mount directories and the GitHub PAT)

@AlexAxthelm you ok with those changes?

Reply:
> * is it necessary/desirable to (explicitly) include the Dockerfile inside the Docker image?

Hmm. I don't know. Try taking it out and see if it builds?

> * I believe it is necessary to include the `.env` file (because the desired config set is specified there, not just for the mount directories and the GitHub PAT)

Good catch.